src/main/resources/assets/galacticraft-rewoven/lang/en_us.json

```diff
@@ -319,6 +319,7 @@
   "item.galacticraft-rewoven.evolved_zombie_spawn_egg": "Evolved Zombie Spawn Egg",
   "item.galacticraft-rewoven.throwable_meteor_chunk": "Throwable Meteor Chunk",
   "item.galacticraft-rewoven.hot_throwable_meteor_chunk": "Hot Throwable Meteor Chunk",
+  "item.galacticraft-rewoven.emergency_kit": "Emergency Space Kit",
```

Comment:
Maybe just `Emergency Kit`? (What does GC4 say)

Reply:
GC4 actually says "Space Emergency Kit"
https://github.com/micdoodle8/Galacticraft/blob/9f80b35bc06e9942e4e2b646bd91710a3f2586ff/src/main/resources/assets/galacticraftcore/lang/en_US.lang#L469

Do we plan on sticking with GC4's naming convention or shorten it to `Emergency Kit`

Reply:
`Emergency Kit` feels more natural to me

Reply:
+1 for Emergency Kit